src/Stateless/Reflection/StateInfo.cs

```diff
@@ -130,12 +122,7 @@ public class StateInfo
         /// <summary>
         /// Transitions defined for this state.
         /// </summary>
-        public IEnumerable<TransitionInfo> Transitions { get; private set; }
```

Comment:
Could this remain, but simply return `FixedTransitions.Concat(DynamicTransitions)`? In .NET reflection, we have this pattern in a few places e.g. https://msdn.microsoft.com/en-us/library/system.reflection.typeinfo.declaredmembers(v=vs.110).aspx)